docs/core/profiles.md

```diff
@@ -151,96 +153,112 @@ can be influenced by the different fields of such a custom model.
+can reduce the speed of every road segment that has the value MOTORWAY for the category 'road_class' to fifty percent of 
 the default speed that is normally used by the base vehicle for this road class:
 ```yaml
 speed_factor:
-  road_class: {motorway: 0.5}
-```  
-Note that `road_class: {motorway: 0.5}` is an alternative YAML notation that is equivalent to:
+  road_class == MOTORWAY: 0.5
+```
+
+You can also change the speed factor for multiple road classes like this
 ```yaml
 speed_factor:
-  road_class:
-    motorway: 0.5
+  road_class == MOTORWAY: 0.5
+  road_class == PRIMARY || road_class == TERTIARY: 0.7
 ```
 
-You can also setup speed factors for multiple road classes like this
+Please note the OR operator `||` that says that if one of the expressions is true the value 0.7 is used.
+
+or use multiple categories to influence the speed factor
 ```yaml
 speed_factor:
-  road_class: {motorway: 0.5, primary: 0.7, tertiary: 0.9}
+  road_class == MOTORWAY: 0.5
+  road_environment == TUNNEL: 0.8
 ```
 
-and use multiple categories to influence the speed factor
+You can use only two operators `==` (equality) and `!=` inequality.
+
+If a road segment matches multiple expressions the speed factor values will be multiplied. For example, here the speed factor of
+a road segment that has `road_class==MOTORWAY` will be `0.5`, the speed factor of a road segment that additionally has
+`road_environment==TUNNEL` will be `0.4` and the speed factor of a road segment that has `road_class==SECONDARY` and
+`road_environment==TUNNEL` will be `0.8`. You can avoid this by using the "first_match" directive:
+
 ```yaml
 speed_factor:
-  road_class: {motorway: 0.5}
-  road_environment: {tunnel: 0.8}
+  first_match:
+    road_class == MOTORWAY: 0.5
+    road_environment == TUNNEL: 0.8
 ```
 
-If an edge matches multiple rules the speed factor values will be multiplied. For example, here the speed factor of 
-a road segment that has `road_class=motorway` will be `0.5`, the speed factor of a road segment that additionally has 
-`road_environment=tunnel` will be `0.4` and the speed factor of a road segment that has `road_class=secondary` and 
-`road_environment=tunnel` will be `0.8`.
+Now even if a road segment fulfills both conditions only the first will be used and no multiplication happens. If you
+know a programming language than the previous statements are "if" clauses and expressions under "first_match" are
+"if-then-else-if" clauses.
 
-Instead of setting the speed factors for certain values you can instead set the speed factors for all *other* values using
-as special key (`"*"`), like this:
+Instead of setting the speed factors for certain values you can instead set the speed factors for all *other* values
+using the unconditional expression `true`:
 ```yaml
-speed_factor: 
-  road_class: {"*": 0.5}
-  road_environment: {tunnel: 0.8, "*": 0.6}
+speed_factor:
+  true: 0.5
```

Comment:
Using `true` as unconditional expression probably makes sense for a programmer who understood that the left side of each entry is a boolean expression, but otherwise its probably hard to understand (compared to something like `default: 0.5`, but of course this requires another pre-processor (which could also convert `|| -> OR`, `&& -> AND`.. Not sure...

Reply:
Also not sure. I only chose the consistent approach, i.e. "Java-alike unconditional".

Reply:
Should we introduce a simple static variable :) ? `boolean DEFAULT = true`

Reply:
Hm let's not introduce more syntactic sugar here until we have finalized the format?

Reply:
Ok, will revert :)